### DISCUSSION TITLE
Localization issue for some languages

### DISCUSSION THREAD

**Discussion by yusasarisoy** [community/community#64410] [Mobile] [answered]
Localizations for German, French and Korean do not work. Here is the example video recording for German:
https://github.com/community/community/assets/30107357/eba186f5-5301-46c0-a86e-caef87ce9ad1
<details>

<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.126.0 (107057637)
Device: iPhone 7 (iOS 15.7.8)
</details>

**Comment by Iron-Ham** [comment, marked as answer]
Hi @yusasarisoy,

Thanks for your feedback! These language localizations are being actively worked on, and shouldn't be available yet. In an upcoming version of the iOS App, the ability to toggle them on will be removed.

Thanks,
Hesham

**Comment by yusasarisoy** [comment]
Thank you for the answer, @Iron-Ham 😊
